Notely/lib/screens/settings.dart

```diff
@@ -0,0 +1,162 @@
+                        color: Color(0xFFF67C0B9),
+                        borderRadius: BorderRadius.circular(4),
+                      ),
+                      child: Theme(
+                        data: ThemeData(
+                          iconTheme: IconThemeData(
+                            color: Colors.white,
+                          ),
+                        ),
+                        child: Column(
+                          mainAxisSize: MainAxisSize.min,
+                          children: List.generate(
+                            icons.length,
+                            (index) {
+                              return GestureDetector(
+                                onTap: () {},
+                                child: Container(
+                                  width: buttonSize.width,
+                                  height: buttonSize.height,
+                                  child: icons[index],
+                                ),
+                              );
+                            },
+                          ),
+                        ),
+                      ),
+                    );
+                  }),
+            ],
+          ),
+
+          SettingsSection(
+              title: 'Privacy',
+              titlePadding: EdgeInsets.all(15),
+              tiles: [
+                SettingsTile.switchTile(
+                  title: 'Private Account',
+                  leading: Icon(Icons.lock),
+                  switchValue: privateAccountValue,
+                  onToggle: (bool toggle) {
+                    //privateAccountValue is changed on user toggle. Important for later.
+                    if (toggle) {
+                      privateAccountValue = true;
+                      leading:
+                      Icon(Icons.ac_unit);
+                    } else {
+                      leading:
+                      Icon(Icons.local_airport_sharp);
+                      privateAccountValue = false;
+                    }
+                  },
+                ),
+              ])
+        ],
+      ),
+      //Here: I'm trying to display the profile picture above the settings page. Still working...
+      persistentFooterButtons: [
+        Container(
+          width: 100,
+          height: 100,
+          decoration: BoxDecoration(
+              border: Border.all(
+                  width: 4, color: Theme.of(context).scaffoldBackgroundColor),
+              boxShadow: [
+                BoxShadow(
+                    spreadRadius: 2,
+                    blurRadius: 10,
+                    color: Colors.black.withOpacity(0.1),
+                    offset: Offset(0, 10))
+              ],
+              shape: BoxShape.circle,
+              image: DecorationImage(
+                  fit: BoxFit.cover,
+                  image: NetworkImage(
+                    "https://cdn2.iconfinder.com/data/icons/facebook-51/32/FACEBOOK_LINE-01-512.png",
+                  ))),
```

Comment:
To move this above the menu, drag it above all of the code. Center with padding.